sentry-core/src/test/java/io/sentry/core/DsnTest.kt

```diff
@@ -11,7 +11,7 @@ class DsnTest {
     fun `dsn parsed with path, sets all properties`() {
         val dsn = Dsn("https://publicKey:secretKey@host/path/id")
 
-        assertEquals("https://host/path/api/id/store/", dsn.sentryUri.toURL().toString())
+        assertEquals("https://host/path/api/id", dsn.sentryUri.toURL().toString())
```

Comment:
This makes me realize this is now a breaking change since if one relies on "DSN" to make requests, that'll be broken.

Reply:
only if they have their own `Transport` which is discouraged, but possible anyway.
I'll point this out on the release notes, good catch.